File: android/src/main/java/au/com/crazybean/mobilex/kurir/modules/signup/SignupView.kt
package au.com.crazybean.mobilex.kurir.modules.signup

import au.com.crazybean.mobilex.kurir.modules.base.View

interface SignupView : View {
    fun showSignup()
    fun showLogin()
    fun showVerify()
    fun showProfile()
    fun showCodeError()
    fun showExists()
    fun showError()
    fun showDashboard()
}

File: ios/Kurir/modules/signup/SignupView.swift
//
//  SignupView.swift
//  Kurir
//
//  Created by Loren on 9/9/19.
//  Copyright © 2019 Crazybean Studio. All rights reserved.
//

import Mobilex

protocol SignupView {
    func showSignup()
    func showLogin()
    func showVerify(user: User?)
    func showExists()
    func showError()
    func showSpinner()
    func hideSpinner()
}
